packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -107,47 +95,47 @@ When a specific client/IP/device identifier combination exceeds either the 60 re
 
 > **Note:** For all orgs created after September 2020, this feature is automatically set to the **Enforce and log per client (recommended)** mode.
 
-To configure client-based rate limiting for existing orgs:
+To configure the client-based rate limit for existing orgs:
 
-1. From the Admin Console select **Settings** > **Account**, and then scroll down to the **Client-based rate limiting** section.
+1. From the Admin Console select **Settings** > **Account**, and then scroll down to the **Client-based rate limit** section.
```

Comment:
This actually appears as "Client-based rate limiting" in the product, so maybe we should keep it as is. 
![image](https://user-images.githubusercontent.com/11844341/110867844-51829500-827c-11eb-8f1a-301ccb1455b2.png)

Reply:
Fixed.